_source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md

```diff
@@ -261,26 +261,28 @@ Here's how it works:
 
 (If this is all new to you, you can [learn more about OAuth here](https://developer.okta.com/blog/2017/06/21/what-the-heck-is-oauth).)
 
-You probably don't want to build an OAuth 2.0 authorization server yourself, so for this tutorial, you will sign up for a [forever-free Okta developer account](https://developer.okta.com/signup/) (or sign in to `{yourOktaDomain}` if you already have an account).
+{% include setup/cli.md type="service" %}
 
 ## Register the Client Application for Your REST API
 
-Once you're signed in to Okta, register your client application.
+Once the Okta CLI is installed, register your client application.
 
-* In the top menu, click on **Applications**
-* Click on **Add Application**
-* Select **Service** and click **Next**
-* Enter `Postman` for the **Name** and click **Done**
+* Run `okta apps create`
+* Enter `Postman` for the **Name** and press enter
+* Choose `4` for the app type to create a `Service` app
+* Choose `5` to create the app using a framework of `Other`
+* Select the `default` Authorization Server and press enter  
 
-Note the **Client ID** and the **Client Secret**. This is the username and password that our client application (Postman in our case) need to send to the authorization server to request an access token. You can make note these credentials now or you can come back and get them later when you need them.
+Run `cat .okta.env` (or type `.okta.env` on Windows) to see the issuer and credentials for your app. Note the **Client ID** and **Client Secret**. This is the username and password that our client application (Postman in our case) need to send to the authorization server to request an access token. You can make note these credentials now or you can come back and get them later when you need them.
```

Comment:
Ideally, all these steps can be removed and the include can print out all of this. I'll take a look and see if this is possible w/o too much trouble.

Reply:
Oh yeah the include does cover a lot. heh. Sorry my first time using. Should be good now.